### ISSUE TITLE
Fix S1075 FP: Do not raise an issue for XML namespace URIs

### ISSUE THREAD

**Issue by mary-georgiou-sonarsource** [SonarSource/sonar-dotnet#8967]
### Description

The syntax for such an XML namespace is a URI. but it is not used to access any network resource. As such it is not subject to the problems described in S1075.

### Repro steps

```csharp
using System.Xml;

private static XmlNamespaceManager CreateNamespaceManager()
{
    var namespaceManager = new XmlNamespaceManager(new NameTable());   // FP
    namespaceManager.AddNamespace("ffc", "http://www.redacted.com/namespaces/ffc"); // FP
    namespaceManager.AddNamespace("dex", "http://www.redacted.com/namespaces/dex"); // FP
    return namespaceManager;
}
```

Follow this past issue on which namespaces should be covered: https://github.com/SonarSource/sonar-dotnet/issues/6223

**Comment by pavel-mikula-sonarsource** [comment]
Internal ticket [NET-1601](https://sonarsource.atlassian.net/browse/NET-1601)

[NET-1601]: https://sonarsource.atlassian.net/browse/NET-1601?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
